src/data/mystery-encounters/encounters/delibirdy-encounter.ts

```diff
@@ -153,12 +153,7 @@ export const DelibirdyEncounter: MysteryEncounter = MysteryEncounterBuilder.with
           const shellBell = generateModifierType(modifierTypes.SHELL_BELL) as PokemonHeldItemModifierType;
           await applyModifierTypeToPlayerPokemon(globalScene.getPlayerPokemon()!, shellBell);
           globalScene.audioManager.playSound("item_fanfare");
-          await showEncounterText(
-            i18next.t("battle:rewardGain", { modifierName: shellBell.name }),
-            null,
-            undefined,
-            true,
-          );
+          await showEncounterText(i18next.t("battle:rewardGain", { modifierName: shellBell.name }), { prompt: true });
```

Comment:
since you said prompt is true by default for showEncounterText in another comment
```suggestion
          await showEncounterText(i18next.t("battle:rewardGain", { modifierName: shellBell.name }));
```